Tema5/positioning.css

```
@@ -0,0 +1,41 @@
+div {
+    border-radius: 10px;
+    border-style: dotted;
+    border-color: rgb(131, 95, 116);
+    text-align: center;
+    text-transform: uppercase;
+}
+.parent {
+    background-color:rgb(187, 201, 184);
+    border-width: 5px;
+    height: 280px;
+    width: 750px;
+}
+.child {
+    border-width: 3px;
+    height: 230px;
+    width: 300px;
+
+}
+
+.child1 {
+    background-color:rgb(81, 141, 138);
+    position:absolute;
+    top:12px;
```

Avand postion setat pe parinte, acum copii se vor alinia cum trebuie relativ la el
```suggestion
    top:0;
```